assets/components/molecules/basic-page-teaser/basic-page-teaser.scss

```diff
@@ -3,53 +3,141 @@
 .basic-page-teaser {
   position: relative;
 
+  @extend .col-12;
+
+  @include media-breakpoint-up(md) {
+    display: flex;
+    flex-flow: row nobreak;
+  }
+
+  @include media-breakpoint-up(xl) {
+    display: block;
+    padding: 0;
+  }
+
   img {
     width: 100%;
   }
 
+  picture,
+  .basic-page-teaser-text {
+    @include media-breakpoint-up(md) {
+      flex: 0 0 50%;
+    }
+  }
+
+  picture {
+    @include media-breakpoint-up(md) {
+      position: relative;
+      overflow: hidden;
+
+      img {
+        position: absolute;
+        top: 0;
+        left: -15vw;
```

Comment:
Why `15` ?

Reply:
It's an arbitrary value. I changed it for 10% to make it more understandable. The idea is to keep the main focus of the picture visible on talbet (which is in the middle left of the 16/9 image), because the design crops the image